plugins/python/uwsgi_python.h

```diff
@@ -32,6 +32,7 @@
 
 #if PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION < 7
 #define HAS_NOT_PyOS_AfterFork_Child
+#define UWSGI_SHOULD_CALL_PYEVAL_INITTHREADS
```

Comment:
wrong ifedefery, this should be (PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION < 7) || PY_MAJOR_VERSION < 3